### PR TITLE
Issue 8605: "show less" now displays "show less"

### LINKED ISSUE #8605: "show less" on Frio forums sidebar is "show more"
### Steps to reproduce the problem

 - Join many forums (or follow many AP groups).
 - Click "show more" in the sidebar.
 - More forums show.

### Expected behavior

 - More forums show.
 - Bottom link now "show less".
 - Clicking it shows less.

### Actual behavior

 - More forums show.
 - Bottom link says "show more".
 - Clicking it shows less.

### Friendica version you encountered the problem

```
This is Friendica, version 2020.06-dev that is running at the web location https://libranet.de. The database version is 1346, the post update version is 1346.
```

Theme: `Frio`

### PR DESCRIPTION
**Pull request by annando**
Fixes https://github.com/friendica/friendica/issues/8605

### CODE REVIEW

File: view/templates/widget_forumlist.tpl [hunk 1]
```diff
@@ -2,12 +2,12 @@
 function showHideForumlist() {
 	if( $("li[id^='forum-widget-entry-extended-']").is(':visible')) {
 		$("li[id^='forum-widget-entry-extended-']").hide();
-		$("li#forum-widget-collapse").html(window.showMore);
```
Comment (MrPetovan) [inline view/templates/widget_forumlist.tpl:5]: Is this global variable used anywhere else? If not, can you please remove its initialization?

Comment (annando) [reply view/templates/widget_forumlist.tpl:5]: I didn't even knew that this had been some variable, nor do I know where it is defined :-)

Comment (annando) [reply view/templates/widget_forumlist.tpl:5]: searching for this only resulted in finding just another place where it hadn't worked :-)

File: src/Content/Widget.php [hunk 2]
```diff
@@ -516,6 +516,7 @@ public static function postedByYear(string $url, int $uid, bool $wall)
 			'$cutoff' => $cutoff,
 			'$url' => $url,
 			'$dates' => $ret,
+			'$showless'	=> DI::l10n()->t('show less'),
```
Comment (AlfredSK) [inline src/Content/Widget.php]: This looks like a wrong indentation*. At least it is different from the other lines.
*) I mean the spaces/tab.

File: view/templates/widget_forumlist.tpl (hunk 1 above)
Comment (MrPetovan) [reply view/templates/widget_forumlist.tpl:5]: How did this ever work?

**Review by MrPetovan — APPROVED** [review]

File: view/templates/widget_forumlist.tpl (hunk 1 above)
Comment (annando) [reply view/templates/widget_forumlist.tpl:5]: Possibly because it was empty it never replaced the content and so the initial content was used?

**Pull request merged**